notebooks/1_spikesorting.ipynb

```diff
@@ -4,26 +4,16 @@
    "cell_type": "markdown",
```

Comment:
<p>I'm tempted to cut off the tutorial here and replace it with a link to the sortingview workspace where the user can visualize the results of sorting. The reason is the stuff including metric computation and automatic curation are still under heavy development and would likely change within a couple of weeks, at which point we would need to write another tutorial. what do you think?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LorenFrankLab/nwb_datajoint/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='69'/>

Reply:
<p>I think that's ok! Will what's reflected in this tutorial not be changed in a couple of weeks?</p>